x-pack/plugins/security_solution/server/usage/detections/rules/get_metrics.ts

```diff
@@ -108,6 +122,7 @@ export const getRuleMetrics = async ({
     return {
       detection_rule_detail: elasticRuleObjects,
       detection_rule_usage: rulesUsage,
+      detection_rule_status: eventLogMetricsTypeStatus,
     };
   } catch (e) {
     // ignore failure, usage will be zeroed. We use debug mode to not unnecessarily worry users as this will not effect them.
```

Comment:
How do you know that a failure occurred in the actual telemetry? On failures we're returning empty values?